backend/frontend/views.py

```diff
@@ -113,14 +121,34 @@ def can_view_user_board(logged_in_user):
         checker.prefetch_perms(viewed_user_teams)
         for view_permission in Team.PERMISSION_VIEW:
             if any(
-                (checker.has_perm(view_permission, team)
-                 for team in viewed_user_teams)
+                (checker.has_perm(view_permission, team) for team in viewed_user_teams)
             ):
                 return True
 
     return False
 
 
+def user_can_start_card(logged_in_user):
```

Comment:
This is strange. There is a function that does this already:

AgileCard.request_user_can_start 

Why the duplication?

Reply:
I now see that it is a permission thing. But the code still isn't dry. It also needs a better naming convention.